shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/metadata/impl/StandardMetaDataContexts.java

```diff
@@ -67,6 +67,16 @@ public StandardMetaDataContexts(final Map<String, ShardingSphereMetaData> metaDa
         this.props = props;
         stateContext = new StateContext();
     }
+
+    public StandardMetaDataContexts(final Map<String, ShardingSphereMetaData> metaDataMap, final ShardingSphereRuleMetaData globalRuleMetaData,
```

Comment:
So the old constructor is supposed to remove?

Reply:
In principle, yes, but I'll get rid of the old constructors as soon as I make sure the PR works. I'll add todo to the old constructor later, what do you think?